app/views/find/pages/privacy.html.erb

```diff
@@ -1,23 +1,39 @@
-<% content_for :page_title, "Privacy policy" %>
+<% content_for :page_title, "Privacy notice" %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Privacy policy</h1>
+    <h1 class="govuk-heading-xl">Privacy notice</h1>
```

Comment:
Did we mean to change this to `notice` or the full heading as per the doc? Also should the link still be `privacy policy` in the footer etc? @Kateharries

Reply:
Clarified with Victoria offline